src/components/molecules/VNavigation.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+  <nav class="nav">
+    <ul class="nav__ul">
+      <li v-for="menu in menus" class="nav__li" :key="menu.id">
+        <router-link :to="menu.pathUrl" exact>
+          <component :is="menu.iconComponent" class="nav__icon" />
+          <div
+            :class="{ 'nav__li--is-active': $route.path === menu.pathUrl }"
+          ></div>
+        </router-link>
+      </li>
+    </ul>
+  </nav>
+</template>
+
+<script>
+import {
+  VSmartHomeIcon,
+  VCalendarTimeIcon,
+  VChartAreaLineIcon,
+} from "../atoms/IconComponents";
+
+export default {
+  name: "VNavigation",
```

Comment:
What about VBottomNavBar ?

Reply:
I think im gonna leave it that way since I only have one navigation here.
But when there are changes or maybe new navigations added to this application in the future, I will change its name to be more specific

Reply:
Ok then